connector/connect/src/main/scala/org/apache/spark/sql/connect/planner/SparkConnectPlanner.scala

```diff
@@ -39,14 +46,17 @@ final case class InvalidPlanInput(
     private val cause: Throwable = None.orNull)
     extends Exception(message, cause)
 
-class SparkConnectPlanner(plan: proto.Relation, session: SparkSession) {
+final case class InvalidCommandInput(
```

Comment:
Not related to this PR, but today we prefer to use error classes to differentiate different errors, instead of using different java exception classes.

Reply:
Agreed. I should adopt and standardize errors in Connect soon.